week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,102 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+console.clear();
+let shakingBallAnswer;
+
+function shakeBall() {
+  let shakingBall = Math.round(Math.random() * 20);
+  switch (shakingBall) {
```

Comment:
Nice, you've discovered `switch` statements! How could you make this code more succinct using arrays? :)